api/coreservice.go

```diff
@@ -1230,6 +1233,31 @@ func (core *coreService) getBlockByHeight(height uint64) (*apitypes.BlockWithRec
 	}, nil
 }
 
+func (core *coreService) BlobSidecarsByHeight(height uint64) ([]*apitypes.BlobSidecarResult, error) {
+	header, err := core.bc.BlockHeaderByHeight(height)
+	if err != nil {
+		return nil, err
+	}
+	res := make([]*apitypes.BlobSidecarResult, 0)
+	blobs, txIndexes, txHashes := core.getBlobSidecars(height)
+	blkHash := header.HashBlock()
+	for i, blob := range blobs {
+		res = append(res, &apitypes.BlobSidecarResult{
+			BlobSidecar: blob,
+			BlockNumber: height,
+			BlockHash:   common.BytesToHash(blkHash[:]),
+			TxIndex:     txIndexes[i],
+			TxHash:      common.BytesToHash(txHashes[i][:]),
+		})
+	}
+	return res, nil
+}
+
+func (core *coreService) getBlobSidecars(height uint64) (blobs []any, txIndexes []uint64, txHashes []hash.Hash256) {
```

Comment:
this func should only return `blobs`
`txIndexes, txHashes` should come from core's other API funcs

Reply:
also txIndexes --> txIndices

Reply:
fixed